### ISSUE TITLE
PyTorch integration

### ISSUE THREAD

**Issue by tillmo** [cherryfunk/mULLER#2]
- Add (optional, via subclass) conversion from `torch.Tensor ` to TΩ to `Aggr2SGrpBLat`, for distribution example: softmax
- add an (optional, via subclass) second `Aggr2SGrpBLat` that is differentiable in `torch` 
- make monad operations differentiable in `torch`
- interpretations should consist of two standard interpretaions plus embedding functions. The second interpretation is on `torch.Tensor `and takes torch models of type `torch.nn.Module` as intepretations for monadic functions and predicates.
  - embedding functions map the first interpretation (of a sort) to the torch interpretation
